### DISCUSSION TITLE
Image pull secret for Mia-Platform plugins

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#149] [Q&A] [answered]
Hi!

I have an on-premise installation of Mia-Platform and all the `*.deployment.yml` files related to Mia-Platform plugins, get generated with the following piece of configuration:

```
spec:
      imagePullSecrets:
        - name: nexus-gcloud
```

This forces me to name the pull secret for mia-platform.nexus.eu `nexus-gcloud`.
Is there a way to change this name so that I can follow my naming convention for Kubernetes secrets?

Thanks in advance!

**Comment by fredmaggiowski** [comment, marked as answer]
Hi, You can configure custom docker image pull secrets from the installation helm chart using the `imagePullSecrets` array

**Comment by davidebianchi** [comment]
We should add this configuration from the UI of the Console: both the image pull secret definition that the mount of the secret (maube with a default secret to mount, and the possibility to define for every service more than one pull secret).

**Reply by davidebianchi** [reply]
@albertotessarotto FYI

**Reply by lucascanna** [reply]
I agree with you @davidebianchi. Having this feature in the Microservices section would be great!
